modules/farm_fd2/src/entrypoints/direct_seeding/lib.submitError.unit.cy.js

```diff
@@ -195,4 +195,55 @@ describe('Error when submitting using the direct_seeding lib.', () => {
         })
     );
   });
+
```

Comment:
I don't think we need this test because the case of attempting to delete those three logs is covered by the first test above.